airbyte-ci/connectors/live-tests/README.md

```diff
@@ -0,0 +1,317 @@
+--connector-image=airbyte/source-pokeapi:dev \
+--connector-image=airbyte/source-pokeapi:latest \
+--connection-id=<CONNECTION-ID>
+```
+
+##### Consuming `http_dump.mitm`
+You can install [`mitmproxy`](https://mitmproxy.org/):
+```bash
+pipx install mitmproxy
+```
+
+And run:
+```bash
+mitmweb --rfile=http_dump.mitm
+```
+
+## Regression tests
+We created a regression test suite to run tests to compare the outputs of connector commands on different versions of the same connector. 
+
+You can run the existing test suites with the following command:
+
+#### With local connection objects (`config.json`, `catalog.json`, `state.json`)
+```bash
+poetry run pytest src/live_tests/regression_tests \ 
+--connector-image=airbyte/source-faker \
+ --config-path=<path-to-config-path> \
+ --catalog-path=<path-to-catalog-path> \
+ --target-version=dev \
+ --control-version=latest
+ --pr-url=<PR-URL> # The URL of the PR you are testing
+```
+
+#### Using a live connection
+The live connection objects will be fetched.
+
+```bash
+ poetry run pytest src/live_tests/regression_tests \
+ --connector-image=airbyte/source-faker \
+ --connection-id=<CONNECTION-ID> \
+ --target-version=dev \
+ --control-version=latest 
+  --pr-url=<PR-URL> # The URL of the PR you are testing
+ ```
+
+You can also pass local connection objects path to override the live connection objects with `--config-path`, `--state-path` or `--catalog-path`.
+
+#### Test artifacts
+The test suite run will produce test artifacts in the `/tmp/regression_tests_artifacts/` folder.
+**They will get cleared after each test run on prompt exit. Please do not copy them elsewhere in your filesystem as they contain sensitive data that are not meant to be stored outside of your debugging session!**
+
+##### Artifacts types
+* `report.html`: A report of the test run.
+* `stdout.log`: The collected standard output following the command execution
+* `stderr.log`: The collected standard error following the command execution
+* `http_dump.mitm`: An `mitmproxy` http stream log. Can be consumed with `mitmweb` (version `>=10`) for debugging.
+* `http_dump.har`: An `mitmproxy` http stream log in HAR format (a JSON encoded version of the mitm dump).
+* `airbyte_messages`: A directory containing `.jsonl` files for each message type (logs, records, traces, controls, states etc.) produced by the connector.
+* `duck.db`: A DuckDB database containing the messages produced by the connector.
+* `dagger.log`: The log of the Dagger session, useful for debugging errors unrelated to the tests.
+
+**Tests can also write specific artifacts like diffs under a directory named after the test function.**
+
+
+```
+/tmp/regression_tests_artifacts
+└── session_1710754231
+    |── report.html
+    ├── command_execution_artifacts
+    │   └── source-orb
+    │       ├── check
+    │       │   ├── dev
+    │       │   │   ├── airbyte_messages
+    │       │   │   │   ├── connection_status.jsonl
+    │       │   │   │   ├── duck.db
```

Comment:
Nit: I forgot to update the readme, this file is now under the session dir

Reply:
Updated.